docs/Datamodel/DSL/README.md

```diff
@@ -0,0 +1,100 @@
+Nexus DSL provides the framework and syntax for declaring Nexus datamodel nodes and types using Golang syntax.
```

Comment:
we've got DSL overview also in compiler directory, maybe we should move it to docs dir as well?
cc @xmen4xp

Reply:
moved the doc reference to compiler